lms/djangoapps/student_account/views.py

```diff
@@ -71,7 +71,7 @@ def login_and_registration_form(request, initial_mode="login"):
     # We need to do this for now to support existing themes.
     # Microsites can use the new logistration page by setting
     # 'ENABLE_COMBINED_LOGIN_REGISTRATION' in their microsites configuration file.
-    if microsite.is_request_in_microsite() and not microsite.get_value('ENABLE_COMBINED_LOGIN_REGISTRATION', False):
+    if is_request_in_themed_site() and not get_themed_value('ENABLE_COMBINED_LOGIN_REGISTRATION', False):
```

Comment:
Should the comment above this be generalized to "themed sites" (instead of mentioning microsites explicitly)?